static/js/apps/nl_interface/debug_info.tsx

```diff
@@ -86,6 +98,26 @@ const matchScoresElement = (svScores: SVScores): JSX.Element => {
   );
 };
 
+function getFeedbackLink(query: string, queryChain: string[]): string {
+  const paramMap = {
+    [QUERY_PARAM_PREFIX]: query,
+    [SOURCE_PARAM_PREFIX]: window.location.toString(),
+    [VERSION_PARAM_PREFIX]:
+      document.getElementById("metadata").dataset.websiteHash || "",
+    [QUERY_CHAIN_PARAM_PREFIX]: JSON.stringify(queryChain),
+  };
+  let link = FEEDBACK_LINK;
+  Object.keys(paramMap).forEach((prefix) => {
+    const value = paramMap[prefix];
+    if (!value) {
+      return;
+    }
+    // when prefilling google forms, need to replace space with "+"
+    link += `${prefix}${value.replaceAll(" ", "+")}`;
```

Comment:
consider calling encodeURI instead, which would also take care of other html special chars

Reply:
done